src/api_routes_websocket.rs

```diff
@@ -299,7 +302,7 @@ pub async fn match_websocket_operation_crud(
   context: LemmyContext,
   id: ConnectionId,
   op: UserOperationCrud,
-  data: &str,
+  data: &Value,
```

Comment:
Cant you pass this by value (without `&`) and avoid using clone?

Reply:
Got it, using `.cloned()` worked.

Reply:
Cloned is the same as clone, it still copies data around unnecessarily.